controllers/updates/views/view-helpers.njk

```diff
@@ -76,3 +76,26 @@
 
     {% endcall %}
 {% endmacro %}
+
+
+{% macro relatedProgrammes(entry) %}
```

Comment:
Maybe make this into a first-class component in `components/`?

Reply:
Done.